index.bs

```diff
@@ -2302,6 +2349,93 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
 Issue: Invoke [=parse summary buckets=] and [=parse summary window operator=]
 from this algorithm.
 
+To <dfn>parse max event states</dfn> from a [=map=] |map|,
```

Comment:
`max_event_states` can be available for both event and navigation ones. it represents the max event states for the fake views in the attack vector.

Reply:
Which attack vector is that? If this proposal is either exacerbating an existing privacy issue or introducing a new one, we should probably note it in https://wicg.github.io/attribution-reporting-api/#privacy-considerations.

I'm also not familiar with the concept of a fake view, since the API in its current form never attempts to associate an event source with an actual user action such as a view. So to me, there's no such thing as a fake view, since there are no views at all. We should non-normatively elaborate on this concern in the spec if it is relevant to this proposal.

Reply:
Synced offline, will add non-normative section.